modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -1253,8 +1249,13 @@ public void createApplication(Application.Name applicationName) {
 
             Map<Application.Name, Set<String>> result = new HashMap<>();
             for (ListenableFuture<Result<ExperimentTagsByApplication>> future : futures) {
-                ExperimentTagsByApplication expTagApplication = future.get().one();
-                result.put(Application.Name.valueOf(expTagApplication.getAppName()), expTagApplication.getTags());
+                List<ExperimentTagsByApplication> expTagApplication = future.get().all();
+
+                Set<String> allTagsForApplication = new TreeSet<>();
```

Comment:
@AndreaSuckro That is what I was asking! :) Whether we need the sorting order anywhere, be it the backend or the UI. Now, as you said that is required on the UI side, it makes sense to order it here.